py4DSTEM/io/legacy/read_utils.py

```diff
@@ -100,7 +97,8 @@ def get_N_dataobjects(filepath, topgroup="4DSTEM_experiment"):
         N_pla = len(f[topgroup]["data/pointlistarrays"].keys())
         try:
             N_coords = len(f[topgroup]["data/coordinates"].keys())
-        except:
+        # TODO work out what exception will be raised ValueError, AttributeError, BS thinks KeyError
```

Comment:
TODO

Reply:
I can remove

Reply:
Actually, I'm going to leave this as a TODO I don't want to break this and I'm not familiar enough with this part to test